lib/RGBot.js

```diff
@@ -1852,7 +1904,14 @@ class RGBot extends EventEmitter {
         return result
     }
 
-
+    /**
```

Comment:
I would add doc to this and `waitForTicks` explaining why you would use one vs. the other

Reply:
Added!